classes/AddressFormat.php

```diff
@@ -284,9 +298,9 @@ protected static function _setOriginalDisplayFormat(&$formattedValueList, $curre
         }
     }
 
-    /*
-    ** Cleaned the layout set by the user
-    */
+    /**
```

Comment:
note this second asterisk is not required in PSR, this is only useful when you want to work with Doctrine and Symfony annotations :)